### PR TITLE
Branch add library functions in model

### PR DESCRIPTION
**Pull request by ashleyclx**
Add helper functions and update borrow

Part of #136

### CODE REVIEW

File: src/main/java/seedu/address/model/library/Library.java [hunk 1]
```diff
@@ -134,6 +134,18 @@ public boolean hasThreshold(Threshold threshold) {
         return this.threshold.equals(threshold);
     }
 
+    public boolean hasBookInLibrary(Book book) {
+        return bookList.contains(book);
+    }
+
+    public Book popBookFromLibrary(Book book) {
```
Comment (Darkarche3) [inline src/main/java/seedu/address/model/library/Library.java:141]: Missing Javadoc comment

File: src/main/java/seedu/address/model/library/Threshold.java
Comment (Darkarche3) [inline src/main/java/seedu/address/model/library/Threshold.java:1]: Unused import on line 3

**Review by Darkarche3 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/library/Library.java [hunk 2]
```diff
@@ -134,6 +134,18 @@ public boolean hasThreshold(Threshold threshold) {
...
+        if (hasBookInLibrary(book)) {
+            int i = bookList.indexOf(book);
+            return bookList.remove(i);
+        }
+        return null;
```
Comment (meowwtama) [inline src/main/java/seedu/address/model/library/Library.java:146]: Add javadocs bro

**Pull request merged**